src/app/reporting/reporting.c

```diff
@@ -67,6 +69,24 @@ EmberEventControl emberAfPluginReportingTickEventControl;
 
 EmAfPluginReportVolatileData emAfPluginReportVolatileData[REPORT_TABLE_SIZE];
 
+/** @brief Configured
+ *
+ * This callback is called by the Reporting plugin whenever a reporting entry
+ * is configured, including when entries are deleted or updated. The
+ * application can use this callback for scheduling readings or measurements
+ * based on the minimum and maximum reporting interval for the entry. The
+ * application should return EMBER_ZCL_STATUS_SUCCESS if it can support the
+ * configuration or an error status otherwise. Note: attribute reporting is
+ * required for many clusters and attributes, so rejecting a reporting
+ * configuration may violate ZigBee specifications.
```

Comment:
Just a thought, but do we want to scrub mentions of `Zigbee` from added code/comments or do we view this as more throw away code until we get the CHIP data/device model in place?

Reply:
We have references to Zigbee either as Zigbee or ZCL already in the data model layer. My take is to do a scrub off this later in a separate pass.